src/Hint/Base.hs

```diff
@@ -140,12 +130,6 @@ type GhcErrLogger = GHC.LogAction
 -- | Module names are _not_ filepaths.
 type ModuleName = String
 
-runGhc1 :: MonadInterpreter m => RunGhc1 m a b
-runGhc1 f a = runGhc (f a)
```

Comment:
I must admit I have no idea why this unusual idiom was in there to begin with. Thanks for getting rid of it!

Reply:
Yup! For the record, I removed it in this PR because I added a parameter to `parseDynamicFlags`, so I would've needed to make a `runGhc3`.